app/src/main/java/com/example/playlistmaker/ui/search/view_model/SearchViewModel.kt

```diff
@@ -20,59 +20,57 @@ import com.example.playlistmaker.ui.player.activity.TrackDetailsActivity
 
 class SearchViewModel(private val searchInteractor: SearchInteractor, private val searchHistoryInteractor:SearchHistoryInteractor): ViewModel() {
 
-    private val RESPONSEOKCODE: Int = 200
-
-    private var searchStatusMutable = MutableLiveData<SearchStatuses>()
-    val searchStatus = searchStatusMutable
-    private var trackListMutable = MutableLiveData<MutableList<Track>>()
-    private var searchEditTextMutable = MutableLiveData<String>()
-    val searchEditText = searchEditTextMutable
-    private var trackHistoryListMutable = MutableLiveData<MutableList<Track>>()
-    private var isShowHistoryListMutable = MutableLiveData<Boolean>()
-    val isShowHistoryList = isShowHistoryListMutable
-
-    fun getTrackHistoryList() : LiveData<MutableList<Track>> {
-        return trackHistoryListMutable
+    private var _searchStatusMutable = MutableLiveData<SearchStatuses>()
```

Comment:
нейминг + val еще нужно поправить
```suggestion
    private val _searchStatus = MutableLiveData<SearchStatuses>()
```

Reply:
Не понял предложение. У меня же две переменных, одна мутабл, вторая нет. Везде меняю мутабл, а в Активити отдаю неизменяемую.